src/FsFhtw/GameLoop.fs

```diff
@@ -0,0 +1,72 @@
+module GameLoop
+
+open DomainSerializer
+open Domain 
+open Game
+
+let printCards (prefix : string) (cards : Card list) =
+    cards
+    |> List.map (serializeCard) //TODO Serialize Card does not seem to produce expected output in the terminal: CardSuit is not rendered
+    |> List.fold (fun prev curr -> prev + " " + curr) ""
+    |> (fun cardString -> printfn "%s%s" prefix cardString)
+
+let generatePlayer(deck : ShuffledDeck) (communityCards : CommunityCards) (playerName : string) (position : int) =
+    let playerCards, deck = deck |> dealHoleCards
+    ({
+        name = playerName; 
+        position = position; 
+        hand = createHand communityCards playerCards; 
+        holeCards = playerCards
+    }
+    , deck)
+
+let generateAiPlayers (deck : ShuffledDeck) (communityCards : CommunityCards) (numberOfPlayers : int) =
+    let rec generatePlayerList(deck : ShuffledDeck) (communityCards : CommunityCards) (position : int) (players : Player list) =
+        if position > 0 then
+            let newPlayer, modifiedDeck = generatePlayer deck communityCards ("Player " + position.ToString()) position
+            generatePlayerList modifiedDeck communityCards (position - 1) (newPlayer :: players)
+        else
+            (players, deck)
+    generatePlayerList deck communityCards numberOfPlayers []
+
+let playRound (numberOfPlayers : int) (model : State) : State = 
+
+    let calculateGainOrLoss (players : Player list) (currentPlayer : Player) : int =
+        let winner = evaluateWinner players
+        if winner |> List.exists(fun(player) -> player.name = model.name) then
+            numberOfPlayers * 100 / winner.Length
+        else
+            -1 * 100
+
+    printfn "Shuffling Deck..."
+    let deck = createDeck () |> shuffleDeck
+    
+    printfn "Dealing Community Cards..."
+    let communityCards, deck = deck |> dealCommunityCards
+
+    printfn "Dealing Player Cards..."
+    let aiPlayers, deck = generateAiPlayers deck communityCards numberOfPlayers
+    let player, _ = generatePlayer deck communityCards model.name (numberOfPlayers + 1)
+    let players = player::aiPlayers
+
+    printfn "Results:"
+    printCards "Community Cards: " communityCards
+    players |> List.iter(fun(player) -> printCards (sprintf "%s yielded a %O with" player.name player.hand.rank) player.holeCards)
+    
+    let winner = evaluateWinner players
```

Comment:
wenn wir den winner hier eh schon haben, warum dann in der gainOrLoss Function nochmal auswerten? Wenns einen grund hat wär oben vl ein kommentar fällig